source/reference/minio-mc/mc-mirror.rst

```diff
@@ -191,44 +173,67 @@ Parameters
    
    Overwrites object(s) on the :mc-cmd:`~mc mirror TARGET`.
 
+   For example, consider an active ``mc mirror --overwrite`` synchronizing content from Source to Destination.
+
+   If an object on Source changes, ``my mirror --overwrite`` synchronizes and overwrites any matching file on Destination.
+
+   Without ``--overwrite``, if an object already exists on the Destination, the mirror process fails for the object that exists in both locations.
+
 .. mc-cmd:: --preserve, a
    
 
-   Preserve file system attributes and bucket policy rules of the
-   :mc-cmd:`~mc mirror SOURCE` on the
-   :mc-cmd:`~mc mirror TARGET`.
+   Preserve file system attributes and bucket policy rules of the :mc-cmd:`~mc mirror SOURCE` on the :mc-cmd:`~mc mirror TARGET`.
 
 .. mc-cmd:: --region
    
 
-   Specify the ``string`` region when creating new bucket(s) on the
-   target. 
+   Specify the ``string`` region when creating new bucket(s) on the target. 
 
    Defaults to ``"us-east-1"``.
 
 .. mc-cmd:: --remove
    
 
-   Removes extraneous object(s) on the target. 
+   Removes object(s) on the Target that do not exist on the Source. 
+
+   For example, objects A, B, and C exist on Source.
+   Objects C, D, and E exist on Target.
+
+   When running `mc mirror --remove`, objects A and B copy to Target and objects D and E are removed from the Target.
+   C already exists on both, so does not change on either. 
```

Comment:
Might need to be more specific here - it's all about namespaces, when the expectation might be CRC/hash. So we're only looking to see if the set of namespaces on Source and Destination differ, and trimming the Destination to match.

An object named C exists on both source and destination. `mc mirrror --remove` only checks that "C" exists and does not verify that the contents match.

Might need to verify that is true if you haven't poked it already. I would assume so, because that would make the need for overwrite more clear

Reply:
Adding a paragraph that points to the `--overwrite` flag.